### PR TITLE
#20 Bell Button added

### PR DESCRIPTION
**Pull request by Shahed-RAJOUB**
Done and all Variables are added in env.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,172 +1,180 @@
...
+            to: email,
+        };
+        console.log(mailTo);
+        axios
+            .post("http://localhost:5001/emails/send-email", mailTo)
+            .then((res) => {
+                console.log(res);
+                console.log("Message Sent");
+                setTimeout(() => setDisable(false), 6000);
+                alert("Message Sent");
```
Comment (StefanTirea) [inline src/App.js:51]: Remove alert? I would just change the notify text after click

File: src/App.js [hunk 2]
```diff
@@ -1,172 +1,180 @@
...
+        let mailTo = {
+            to: email,
+        };
+        console.log(mailTo);
+        axios
+            .post("http://localhost:5001/emails/send-email", mailTo)
+            .then((res) => {
+                console.log(res);
+                console.log("Message Sent");
+                setTimeout(() => setDisable(false), 6000);
```
Comment (StefanTirea) [inline src/App.js:50]: `6000` => 6s

**Review by StefanTirea — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (Shahed-RAJOUB) [reply src/App.js:51]: I will keep it to make the User comfortable

File: src/App.js (hunk 2 above)
Comment (Shahed-RAJOUB) [reply src/App.js:50]: thanks
changed

**Pull request merged**